packages/itwinui-css/backstop/tests/table.html

```diff
@@ -939,8 +939,8 @@ <h2>Default</h2>
             </div>
           </div>
 
-          <div class="iui-row iui-selected">
-            <div class="iui-cell iui-slot">
+          <div class="iui-table-row iui-selected">
```

Comment:
Because this is a v1.0 PR we could use `aria-selected` instead of a class modifier.
```suggestion
<div class="iui-table-row" aria-selected="true">
```
You'll need to set `aria-multiselectable="true"` on one of the parents, you can read about it [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected#description).

Reply:
`iui-selected` has been replaced with `aria-selected='true'`.

For tables with selectable rows, the `iui-table-body` element now has `aria-multiselectable='true'`